app/src/main/java/com/example/ithaca_transit_android_v2/states/RouteCard.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.ithaca_transit_android_v2.states
+
+import com.example.ithaca_transit_android_v2.Models.Trip
+import com.example.ithaca_transit_android_v2.models.Coordinate
+import com.example.ithaca_transit_android_v2.models.Location
+import java.util.Date
+
+// Parent Class
+sealed class RouteCard (
+    val trip: Trip = Trip(
+        emptyList(),
+        Location("", Coordinate(0.0,0.0),""),
+        Location("", Coordinate(0.0,0.0),""),
+        false,
+        Date(0),
+        Date(0),
+        0
+    )
+){}
+
+/* Route cards at the bottom of launch screen
```

Comment:
Prefer to have one set of `/*` before you begin the comment, so:

```
/*
 * Route cards at the ...
 */
```

Can you make this change everywhere?